controllers/nodeagent.go

```diff
@@ -47,19 +58,39 @@ var (
 	}
 )
 
-func getFsPvHostPath() string {
-	env := os.Getenv("RESTIC_PV_HOSTPATH")
-	envFs := os.Getenv("FS_PV_HOSTPATH")
-
-	if env != "" {
+// getFsPvHostPath returns the host path for persistent volumes based on the platform type.
+func getFsPvHostPath(platformType string) string {
+	// Check if environment variables are set for host paths
+	if env := os.Getenv(ResticPVHostPathEnvVar); env != "" {
 		return env
 	}
-
-	if envFs != "" {
+	if envFs := os.Getenv(FSPVHostPathEnvVar); envFs != "" {
 		return envFs
 	}
```

Comment:
If there is another change, I would prefer this be rearranged where FS env override restic, not the other way around.

Not blocking.

Reply:
Done, updated !